sorting-algorithms/problem-set-sorting-algorithms.md

```diff
@@ -212,7 +31,7 @@ The insertion sort algorithm is unique because...
 ### !end-challenge
 <!-- prettier-ignore-end -->
 
-<!-- Question 7 -->
+<!-- Question 2 -->
```

Comment:
I'm not a fan of having the question number comments in the markdown. I know this is maintaining the consistency with what's here already, but these just seem to be potential busy work.